src/reanimated2/js-reanimated/JSReanimated.ts

```diff
@@ -14,8 +15,8 @@ export default class JSReanimated extends NativeReanimated {
 
   constructor() {
     super(false);
-    if (process.env.JEST_WORKER_ID) {
-      this.timeProvider = { now: () => Date.now() };
+    if (isJest()) {
+      this.timeProvider = { now: () => global.ReanimatedDataMock.now() };
```

Comment:
return global.ReanimatedDataMock.now();
                                           ^

TypeError: Cannot read property 'now' of undefined

Reply:
I'm getting this error with latest version of reanimated and jest 28. This is where it breaks. Any ideas on how to fix this correctly? @enahum @kkafar